mpi-proxy-split/split_process.cpp

```diff
@@ -471,6 +471,11 @@ getUhVdsoLdAddr()
 void
 updateVdsoLinkmapEntry(void * l_ld_addr)
 {
+  if (l_ld_addr == NULL) {
+    // The lower-half link map did not have a link to "vdso".
```

Comment:
Yes, it is normal -- the issue with -fallow-argument-mismatch.  In face, I separated out the FORTRAN_TESTS target, so that you can now comment that out.  if you comment it out, then you don't have to comment out the `-fallow-argument-mismatch`.  But you're right.  When I get a chance, I want to fix the configure to check for how to compile with Fortran, and what needs to be configured differently.